doc/nrf/gs_modifying.rst

```diff
@@ -181,12 +282,32 @@ Configuring build types
 
 .. build_types_overview_start
 
-Build types enable you to use different sets of configuration options for each board.
-You can create several build type :file:`.conf` files per board and select one of them when building the application.
-This means that you do not have to use one :file:`prj.conf` file for your project and modify it each time to fit your needs.
+The application is defined by a set of configuration files that provide definitions related to software and hardware, which together form the *configuration set* of the application.
+The configuration set specifies which features the application is going to use and how it is going to behave.
+For convenience, you can specify multiple configuration sets to quickly switch the application behavior.
+The most common example is to switch between ``release`` and ``debug`` variants of the application.
+
+In the |NCS|, you can define multiple build types.
+Select the configuration set for building the application by providing a reference to a specific build type file.
+This allows you to use different sets of configuration options for each board.
+
+The :file:`prj.conf` file is the main source for the :ref:`configure_application_sw`, owned by the application.
+You can create additional :file:`conf` files with specific software configuration set by following the name pattern :file:`prj_buildtype.conf`, where *buildtype* is replaced by the name of the build type (see :ref:`gs_modifying_build_types_creating`).
```

Comment:
Shouldn't we be denoting this somehow?

prj_<buildtype>.conf ?

Reply:
We should denote it with italics, but italics are not accepted within the file tag.